homeassistant/components/homewizard/config_flow.py

```diff
@@ -28,6 +28,21 @@ def __init__(self) -> None:
         """Initialize the HomeWizard config flow."""
         self.config: dict[str, str | int] = {}
 
+    async def async_step_import(self, import_config: dict) -> FlowResult:
```

Comment:
What triggers this import step? I'm not seeing it?

Reply:
The custom integration when it is updated, see https://github.com/DCSBL/ha-homewizard-energy/blob/2067090f20ce74a9f8c133e3dcccd40a79ca524d/custom_components/homewizard_energy/__init__.py

https://github.com/DCSBL/ha-homewizard-energy/pull/79